IGCexpansion/CodonGeneconv.py

```diff
@@ -1147,6 +1217,8 @@ def get_IGC_nonsynonymous_contribution(self):
                 return self.Tau_Omega

```

Comment:
Why not `* self.tau` here?

Reply:
Is it that this parameter is the product already?

Reply:
From my understanding, `Tau_Omega` is IGC rate for nonsynonymous change, therefore I recognize it as `self.tau * IGC_Omega`.